src/bench.rs

```diff
@@ -21,32 +22,30 @@ impl BenchCounter {
         ((new.value - old.value) / new.value) * 100.0
     }
 
-    pub fn is_significant(old: &Self, new: &Self, repetitions: u32) -> bool {
-        let f = old;
-        let m = new;
-
-        let half = {
-            let z = get_stat_score_95(2 * repetitions - 2);
-            let n1: f64 = repetitions as f64;
-            let n2: f64 = repetitions as f64;
-            let normer = (1.0 / n1 + 1.0 / n2).sqrt();
-            let numer1 = (n1 - 1.0) * m.variance;
-            let numer2 = (n2 - 1.0) * f.variance;
-            let df = n1 + n2 - 2.0;
-            let sp = ((numer1 + numer2) / df).sqrt();
-            (z * sp * normer) * 100.0 / f.value
-        };
+    /// Perform a t-test with a 95% confidence interval.
+    pub fn is_significant(old: &Self, new: &Self) -> bool {
+        // Shorter names matching how the t-test is often thought.
```

Comment:
what does this mean?

Reply:
should that be "taught"?

Reply:
and maybe just make that "variable names"?

Reply:
The mathematical notation would use things like `x̄` rather than `value` or `s²` instead of `variance`. To make it easier to relate the code that is written to the formal mathematical notation as thought to students, I added a bunch of aliases with an ascii translation of the mathematical symbols, so `x_bar` instead of `x̄` and `s_sqr` instead of `s²`.

Reply:
sure that's fine, I think I'm getting tripped up on "matching" being both a verb and not a verb (e.g. matching colors), so that sentence is confusing. I'd rephrase that as 

```suggestion
        // We use short variable names that match how the t-test is often taught.
```